packages/cli/commands/init.js

```diff
@@ -82,7 +84,7 @@ exports.command = 'init';
 exports.describe = `initialize ${DEFAULT_HUBSPOT_CONFIG_YAML_FILE_NAME} for a HubSpot account`;
 
 exports.handler = async options => {
-  const { auth: authType = PERSONAL_ACCESS_KEY_AUTH_METHOD.value } = options;
+  const { auth: authType = PERSONAL_ACCESS_KEY_AUTH_METHOD.value, c } = options;
   const configPath = getConfigPath();
```

Comment:
What if the path was passed in to `getConfigPath()` like we're doing elsewhere instead of adding `_configPath` check to `getConfigPath()`?

Reply:
If we pass the path to `getConfigPath`, the error `The config file '${configPath}' already exists.` would get logged and the process would exit.

I think it makes sense to look for `_configPath` in `getConfigPath` so that non-standard configs are taken into account. I added a failing test to demonstrate the current behavior, which seems incorrect to me.

![image](https://user-images.githubusercontent.com/6472448/114595694-aa1ec680-9c5c-11eb-9940-76adff466784.png)

Reply:
Makes sense. Perhaps a check against the file existing is the way to go via `fs.existsSync()` instead of the code making the assumption that getting the path is looking for a file under the hood.

Reply:
I'm not seeing where I'm making that assumption. Does getting the set path need to know if there is a file or not?

I don't think we need to add that file check into `getConfigPath`.  It seems to work well without being tied to the file existing.

Reply:
The conditional on line 94 is intended to check if there is a config file at the path and if so to exit with an error. This was working because behind the scenes `getConfigPath()` was looking for the existence of a file instead of the code being written as an explicit check. The new logic would be more accurate and consistent if it was checking for the existence of the file in all cases.

Reply:
Got it -- I added a check for `configFileExists()` prior to returning the `_configPath`